lmdeploy/pytorch/backends/cuda/op_backend.py

```diff
@@ -48,7 +48,11 @@ def get_layer_impl_builder(cls, layer_type: OpType):
             from .activation import TritonSiluAndMulBuilder
             return TritonSiluAndMulBuilder
         elif layer_type == OpType.LinearW4A16:
-            from awq.modules.linear.gemm import AWQ_INSTALLED
+            try:
+                from awq.modules.linear.gemm import awq_ext  # noqa: F401
+                AWQ_INSTALLED = True
+            except Exception:
+                AWQ_INSTALLED = False
```

Comment:
Do we have a warning for awq_ext not installed?

Reply:
Users always complain that the AWQ models run slow. But they may not know it is because of not installed `awq_ext`